### DISCUSSION TITLE
Group Assignment #5 📋

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#25] [Assignments]
#### Due 6/1

For [Group Assignment #5](https://albertkun.github.io/23S-ASIAAM-191A/assignments/week8/group_assignment/), create a rough draft of your website, include a `readme.md`, and be prepared to give a short 2-3 minute presentation with your group members. 

### Submission
Post the link to your GitHub repository and website below.

ex.
**Group Repo:** https://github.com/akochaphum/my-awesome-group
**Group website:** https://akochaphum.github.io/my-awesome-group/index.html

**Comment by chrysanthemma** [comment]
Group Repo: https://github.com/chrysanthemma/foodinsecurity
Group website: https://chrysanthemma.github.io/foodinsecurity/index.html

**Comment by joanneluong** [comment]
Group Repo: https://github.com/joanneluong/AAS191A-SEA
Group website: https://joanneluong.github.io/AAS191A-SEA/index.html

**Comment by jamesfalkenroth** [comment]
Group Repo: https://github.com/jamesfalkenroth/healthcare
Group website: https://jamesfalkenroth.github.io/healthcare/index.html

**Comment by czhangy** [comment]
Group Repo: https://github.com/czhangy/aa191a-inequality
Group website: https://czhangy.github.io/aa191a-inequality/

**Comment by catherinehhu** [comment]
Group repo: https://github.com/catherinehhu/asian-am-191a-project
Group website: https://catherinehhu.github.io/asian-am-191a-project/index.html

**Comment by danigold1020** [comment]
Ocean Pollution: The Oilies
Group Repo: https://danigold1020.github.io/asianam-191a-oceanpollution/
Group Website: https://danigold1020.github.io/asianam-191a-oceanpollution/Project/index.html

**Comment by SageLuong** [comment]
**Westwood Reimagined**
Group Repo: https://github.com/DylanJWan/AA191ATransportation
Group Website: https://dylanjwan.github.io/AA191ATransportation/index.html

**Comment by cassiuscarandang** [comment]
Student Housing 4 All
Group Repo: https://github.com/cassiuscarandang/aashousingandhomelessness?search=1
Group Website: https://cassiuscarandang.github.io/aashousingandhomelessness/

**Comment by stellakangj** [comment]
Group Website: https://trinhqale.github.io/AM191A-group-project/

**Reply by stellakangj** [reply]
Group Repo: https://github.com/trinhqale/AM191A-group-project

**Reply by stellakangj** [reply]
Slides: https://docs.google.com/presentation/d/1LAwZTTuk7Ut8cEvVkNiIFMb4C0LbBt0NiIByxjQf1Dw/edit#slide=id.g22603030741_2_368

**Comment by rebeccaksuen** [comment]
https://rebeccaksuen.github.io/Educational-Attainment/FInal%20Project/

**Reply by rebeccaksuen** [reply]
group repo: https://github.com/rebeccaksuen/Educational-Attainment
